From 0e293f307645492e5eae8640f2da89befd6f53ff Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Mon, 1 Sep 2014 11:34:40 +0000
Subject: [PATCH] Find the right Phonon version

When building against Qt5 version of Phonon look for the right
CMake module.
---
 CMakeLists.txt | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8387d05..e21e39b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,7 +12,15 @@ set(PHONON_GST_VERSION "${PHONON_GST_MAJOR_VERSION}.${PHONON_GST_MINOR_VERSION}.
 
 add_definitions(-DPHONON_GST_VERSION="${PHONON_GST_VERSION}")
 
-find_package(Phonon REQUIRED)
+set(PHONON_LIB_SONAME phonon)
+set(PHONON_LIB_SONAME_CAMEL Phonon)
+
+if(PHONON_BUILD_PHONON4QT5)
+    set(PHONON_LIB_SONAME phonon4qt5)
+    set(PHONON_LIB_SONAME_CAMEL Phonon4Qt5)
+endif(PHONON_BUILD_PHONON4QT5)
+
+find_package(${PHONON_LIB_SONAME_CAMEL} REQUIRED)
 macro_ensure_version("4.7.0" ${PHONON_VERSION} PHONON_FOUND)
 macro_log_feature(PHONON_FOUND "Phonon" "Core Library" "git://anongit.kde.org/phonon.git" TRUE "4.7.0")
 
-- 
1.8.3-rc3

